ext/src/Bigint.c

```diff
@@ -37,17 +46,7 @@ to_double(zval *result, php_driver_numeric *bigint TSRMLS_DC)
 static int
 to_long(zval *result, php_driver_numeric *bigint TSRMLS_DC)
 {
-  if (bigint->data.bigint.value < (cass_int64_t) LONG_MIN) {
```

Comment:
I take back what I said, this is a valid check. On a 32-bit system `long`/`zend_long` is a 32-bit signed value so it can't hold the entire range of bigint.